From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: vanhofen <vanhofen@gmx.de>
Date: Mon, 26 Oct 2026 22:40:21 +0100
Subject: [PATCH] configure.ac: allow autoconf 2.71

Python 3.13+ requires autoconf 2.72 to regenerate configure. The
buildsystem regenerates configure with the build host's autoconf,
and common build hosts (e.g. Debian 12, Ubuntu 24.04) still ship
autoconf 2.71, which handles configure.ac just fine.

Buildroot does not need this, as it uses its own host-autoconf.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 5dbc499..ca37420 100644
--- a/configure.ac
+++ b/configure.ac
@@ -12,7 +12,7 @@ dnl
 # Set VERSION so we only need to edit in one place (i.e., here)
 m4_define([PYTHON_VERSION], [3.14])
 
-AC_PREREQ([2.72])
+AC_PREREQ([2.71])
 
 AC_INIT([python],[PYTHON_VERSION],[https://github.com/python/cpython/issues/])
 
